yt_dlp/extractor/facebook.py

```diff
@@ -395,6 +395,9 @@ def _perform_login(self, username, password):
         try:
             login_results = self._download_webpage(request, None,
                                                    note='Logging in', errnote='unable to fetch login page')
+            if self._html_search_regex(r'(Your Request Couldn.+? be Processed)', login_results, 'request error', default=None) is not None:
+                raise ExtractorError('Failed to perform login request.')
+
             if re.search(r'<form(.*)name="login"(.*)</form>', login_results) is not None:
                 error = self._html_search_regex(
                     r'(?s)<div[^>]+class=(["\']).*?login_error_box.*?\1[^>]*><div[^>]*>.*?</div><div[^>]*>(?P<error>.+?)</div>',
```

Comment:
Not really. Since this is just an out-loud exclamation mark that this method should even be marked as `deprecated` possibly and `ExtractorError` is for that reason not using `expected=True`.

Regarding regex, raw value is this:
```
Your Request Couldn&#039;t be Processed
```
so straight up `text in results` will be probably too explicit and `_html_search_regex(r'(Your Request Couldn\'t be Processed)'`  looks a bit weird, but I guess I can set is as such.